workflow-bot-app/src/test/java/com/symphony/bdk/workflow/UpdateMessageIntegrationTest.java

```diff
@@ -49,10 +49,36 @@ void updateMessageSuccessfull() throws IOException, ProcessingException {
     ArgumentCaptor<Message> messageArgumentCaptor = ArgumentCaptor.forClass(Message.class);
     verify(messageService, times(1)).update(eq(message), messageArgumentCaptor.capture());
     assertThat(messageArgumentCaptor.getValue().getContent()).isEqualTo(content);
+    assertThat(messageArgumentCaptor.getValue().getSilent()).isTrue();
   }
 
   @Test
-  void updateMessageWithTemplateSuccessfull() throws IOException, ProcessingException {
+  void updateMessageWithoutSilentSuccessfully() throws IOException, ProcessingException {
```

Comment:
can we use ParameterizedTest for the two tests? The argument would be the workflow file path and the expected "silent" boolean value.